src/briefcase/commands/new.py

```diff
@@ -525,18 +521,18 @@ def new_app(
     ):
         """Ask questions to generate a new application, and generate a stub project from
         the briefcase-template."""
-        self.input.prompt()
-        self.input.prompt("Let's build a new Briefcase app!")
+        self.console.prompt()
+        self.console.prompt("Let's build a new Briefcase app!")
 
         context = self.build_app_context(project_overrides)
         bootstrap = self.create_bootstrap(context, project_overrides)
         context.update(self.build_gui_context(bootstrap, project_overrides))
 
-        self.input.divider()  # close the prompting section of output
+        self.console.debug()  # close the prompting section of output
```

Comment:
```suggestion
        self.console.divider()  # close the prompting section of output
```

Reply:
The same error exists in convert; I've fixed it there as well.